README.md

```diff
@@ -184,6 +184,11 @@ vim.keymap.set(
   :RustLsp[!] debuggables {args[]}?
   :RustLsp[!] debug {args[]}?
   ```
+  Example:
+  1. Press escape
```

Comment:
This sounds like you're instructing to leave insert mode? What if you're already in normal mode?